components/02-molecules/menu/menu.twig

```diff
@@ -33,7 +33,6 @@
 {% endif %}
 
 {% set menu__attributes = menu__attributes|merge({
-  'id': menu__base_class,
```

Comment:
probably a good idea to remove this and all ID's to be added when needed on a per-menu basis.